HealthSDK/GiniHealthSDK/Sources/GiniHealthSDK/Core/PaymentComponent/PaymentComponentViewModel.swift

```diff
@@ -139,6 +145,23 @@ final class PaymentComponentViewModel {
     }
     
     func tapOnPayInvoiceView() {
+        savePaymentComponentViewUsageStatus()
         delegate?.didTapOnPayInvoice(documentId: documentId)
     }
+    
+    // Function to check if Payment was used at least once
+    func wasPaymentComponentViewUsed() -> Bool {
```

Comment:
[Suggestion]: @razvancapra shall we follow the Apple naming convention and use a name like in `isUserInteractionEnabled` -> `isPaymentComponentViewUsed` or even `isPaymentComponentUsed` ?